 Container krymancer-db-1  Creating
 Container krymancer-db-1  Created
 Container krymancer-api2-1  Creating
 Container krymancer-api1-1  Creating
 Container krymancer-api2-1  Created
 Container krymancer-api1-1  Created
 Container krymancer-nginx-1  Creating
 Container krymancer-nginx-1  Created
Attaching to api1-1, api2-1, db-1, nginx-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
db-1     | selecting default time zone ... Etc/UTC
db-1     | creating configuration files ... ok
db-1     | running bootstrap script ... ok
api1-1   | info: Microsoft.Hosting.Lifetime[14]
api1-1   |       Now listening on: http://localhost:3000
api1-1   | info: Microsoft.Hosting.Lifetime[0]
api1-1   |       Application started. Press Ctrl+C to shut down.
api1-1   | info: Microsoft.Hosting.Lifetime[0]
api1-1   |       Hosting environment: Production
api1-1   | info: Microsoft.Hosting.Lifetime[0]
api1-1   |       Content root path: /app
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
api2-1   | info: Microsoft.Hosting.Lifetime[14]
api2-1   |       Now listening on: http://localhost:3001
api2-1   | info: Microsoft.Hosting.Lifetime[0]
api2-1   |       Application started. Press Ctrl+C to shut down.
api2-1   | info: Microsoft.Hosting.Lifetime[0]
api2-1   |       Hosting environment: Production
api2-1   | info: Microsoft.Hosting.Lifetime[0]
api2-1   |       Content root path: /app
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: /etc/nginx/conf.d/default.conf is not a file or does not exist
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
db-1     | performing post-bootstrap initialization ... ok
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | waiting for server to start....2024-02-21 01:13:48.990 UTC [48] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-21 01:13:48.994 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-21 01:13:49.003 UTC [51] LOG:  database system was shut down at 2024-02-21 01:13:47 UTC
db-1     | 2024-02-21 01:13:49.010 UTC [48] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | INSERT 0 5
db-1     | CREATE FUNCTION
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/postgresql.conf
db-1     | 
db-1     | waiting for server to shut down...2024-02-21 01:13:49.229 UTC [48] LOG:  received fast shutdown request
db-1     | .2024-02-21 01:13:49.231 UTC [48] LOG:  aborting any active transactions
db-1     | 2024-02-21 01:13:49.233 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
db-1     | 2024-02-21 01:13:49.233 UTC [49] LOG:  shutting down
db-1     | 2024-02-21 01:13:49.235 UTC [49] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-21 01:13:49.726 UTC [49] LOG:  checkpoint complete: wrote 939 buffers (5.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.385 s, sync=0.094 s, total=0.494 s; sync files=308, longest=0.072 s, average=0.001 s; distance=4292 kB, estimate=4292 kB; lsn=0/191BC80, redo lsn=0/191BC80
db-1     | 2024-02-21 01:13:49.732 UTC [48] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-02-21 01:13:49.864 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-21 01:13:49.864 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-02-21 01:13:49.864 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-02-21 01:13:49.870 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-21 01:13:49.879 UTC [66] LOG:  database system was shut down at 2024-02-21 01:13:49 UTC
db-1     | 2024-02-21 01:13:49.887 UTC [1] LOG:  database system is ready to accept connections
db-1     | 2024-02-21 01:15:18.591 UTC [174] FATAL:  sorry, too many clients already
api1-1   | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api1-1   |       Connection id "0HN1IF9EGI55O", Request id "0HN1IF9EGI55O:00000001": An unhandled exception was thrown by the application.
api1-1   |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api1-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api1-1   |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.NpgsqlConnection.Open() + 0x28
api1-1   |          at Program.<>c__DisplayClass0_0.<<<Main>$>b__1>d.MoveNext() + 0x14a
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FF60964370F5E176CAF4E62DD6A4081857770DD4FE137DF75793748D018A6ED58__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x41c
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api1-1   |         Exception data:
api1-1   |           Severity: FATAL
api1-1   |           SqlState: 53300
api1-1   |           MessageText: sorry, too many clients already
api1-1   |           File: proc.c
api1-1   |           Line: 351
api1-1   |           Routine: InitProcess
db-1     | 2024-02-21 01:15:18.600 UTC [175] FATAL:  sorry, too many clients already
api1-1   | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api1-1   |       Connection id "0HN1IF9EGI55P", Request id "0HN1IF9EGI55P:00000001": An unhandled exception was thrown by the application.
api1-1   |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api1-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api1-1   |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.NpgsqlConnection.Open() + 0x28
api1-1   |          at Program.<>c__DisplayClass0_0.<<<Main>$>b__1>d.MoveNext() + 0x14a
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FF60964370F5E176CAF4E62DD6A4081857770DD4FE137DF75793748D018A6ED58__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x41c
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api1-1   |         Exception data:
api1-1   |           Severity: FATAL
api1-1   |           SqlState: 53300
api1-1   |           MessageText: sorry, too many clients already
api1-1   |           File: proc.c
api1-1   |           Line: 351
api1-1   |           Routine: InitProcess
db-1     | 2024-02-21 01:15:18.603 UTC [176] FATAL:  sorry, too many clients already
db-1     | 2024-02-21 01:15:18.604 UTC [177] FATAL:  sorry, too many clients already
api1-1   | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api1-1   |       Connection id "0HN1IF9EGI55Q", Request id "0HN1IF9EGI55Q:00000001": An unhandled exception was thrown by the application.
api1-1   |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api1-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api1-1   |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.NpgsqlConnection.Open() + 0x28
api1-1   |          at Program.<>c__DisplayClass0_0.<<<Main>$>b__1>d.MoveNext() + 0x14a
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FF60964370F5E176CAF4E62DD6A4081857770DD4FE137DF75793748D018A6ED58__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x41c
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api1-1   |         Exception data:
api1-1   |           Severity: FATAL
api1-1   |           SqlState: 53300
api1-1   |           MessageText: sorry, too many clients already
api1-1   |           File: proc.c
api1-1   |           Line: 351
api1-1   |           Routine: InitProcess
api2-1   | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api2-1   |       Connection id "0HN1IF9G35JRV", Request id "0HN1IF9G35JRV:00000001": An unhandled exception was thrown by the application.
api2-1   |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api2-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api2-1   |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.NpgsqlConnection.Open() + 0x28
api2-1   |          at Program.<>c__DisplayClass0_0.<<<Main>$>b__1>d.MoveNext() + 0x14a
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FF60964370F5E176CAF4E62DD6A4081857770DD4FE137DF75793748D018A6ED58__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x41c
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api2-1   |         Exception data:
api2-1   |           Severity: FATAL
api2-1   |           SqlState: 53300
api2-1   |           MessageText: sorry, too many clients already
api2-1   |           File: proc.c
api2-1   |           Line: 351
api2-1   |           Routine: InitProcess
db-1     | 2024-02-21 01:15:18.613 UTC [178] FATAL:  sorry, too many clients already
api2-1   | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api2-1   |       Connection id "0HN1IF9G35JS0", Request id "0HN1IF9G35JS0:00000001": An unhandled exception was thrown by the application.
api2-1   |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api2-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api2-1   |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.NpgsqlConnection.Open() + 0x28
api2-1   |          at Program.<>c__DisplayClass0_0.<<<Main>$>b__1>d.MoveNext() + 0x14a
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FF60964370F5E176CAF4E62DD6A4081857770DD4FE137DF75793748D018A6ED58__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x41c
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api2-1   |         Exception data:
api2-1   |           Severity: FATAL
api2-1   |           SqlState: 53300
api2-1   |           MessageText: sorry, too many clients already
api2-1   |           File: proc.c
api2-1   |           Line: 351
api2-1   |           Routine: InitProcess
db-1     | 2024-02-21 01:15:18.616 UTC [179] FATAL:  sorry, too many clients already
api1-1   | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api1-1   |       Connection id "0HN1IF9EGI55R", Request id "0HN1IF9EGI55R:00000001": An unhandled exception was thrown by the application.
api1-1   |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api1-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api1-1   |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Npgsql.NpgsqlConnection.Open() + 0x28
api1-1   |          at Program.<>c__DisplayClass0_0.<<<Main>$>b__1>d.MoveNext() + 0x14a
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FF60964370F5E176CAF4E62DD6A4081857770DD4FE137DF75793748D018A6ED58__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x41c
api1-1   |       --- End of stack trace from previous location ---
api1-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api1-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api1-1   |          at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.<ProcessRequests>d__238`1.MoveNext() + 0x46f
api1-1   |         Exception data:
api1-1   |           Severity: FATAL
api1-1   |           SqlState: 53300
api1-1   |           MessageText: sorry, too many clients already
api1-1   |           File: proc.c
api1-1   |           Line: 351
api1-1   |           Routine: InitProcess
db-1     | 2024-02-21 01:15:18.617 UTC [180] FATAL:  sorry, too many clients already
api2-1   | fail: Microsoft.AspNetCore.Server.Kestrel[13]
api2-1   |       Connection id "0HN1IF9G35JS1", Request id "0HN1IF9G35JS1:00000001": An unhandled exception was thrown by the application.
api2-1   |       Npgsql.PostgresException (0x80004005): 53300: sorry, too many clients already
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<ReadMessageLong>d__233.MoveNext() + 0x995
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.ThrowForFailedGetResult() + 0x16
api2-1   |          at System.Threading.Tasks.Sources.ManualResetValueTaskSourceCore`1.GetResult(Int16) + 0x3e
api2-1   |          at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16) + 0x2c
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<Authenticate>d__0.MoveNext() + 0x1f6
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<<Open>g__OpenCore|213_1>d.MoveNext() + 0x531
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.Internal.NpgsqlConnector.<Open>d__213.MoveNext() + 0x6d9
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.PoolingDataSource.<OpenNewConnector>d__37.MoveNext() + 0x54b
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.PoolingDataSource.<<Get>g__RentAsync|34_0>d.MoveNext() + 0x1c7
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.NpgsqlConnection.<<Open>g__OpenAsync|42_0>d.MoveNext() + 0x3de
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Npgsql.NpgsqlConnection.Open() + 0x28
api2-1   |          at Program.<>c__DisplayClass0_0.<<<Main>$>b__1>d.MoveNext() + 0x14a
api2-1   |       --- End of stack trace from previous location ---
api2-1   |          at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x1c
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0xbe
api2-1   |          at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task, ConfigureAwaitOptions) + 0x4e
api2-1   |          at Microsoft.AspNetCore.Http.Generated.<GeneratedRouteBuilderExtensions_g>FF60964370F5E176CAF4E62DD6A4081857770DD4FE137DF75793748D018A6ED58__GeneratedRouteBuilderExtensionsCore.<>c__DisplayClass3_0.<<MapPost0>g__RequestHandler|4>d.MoveNext() + 0x41c
api2-1   |       --- End of stack trace from previous location ---
